elasticsearch/Dockerfile

```diff
@@ -1,12 +1,3 @@
-#oit## This file is managed by the OpenShift Image Tool: https://github.com/openshift/enterprise-images
```

Comment:
I would add all this generated bits back in

Reply:
@jcantrill why?

Reply:
I defer to @adammhaile  here.  I believe the downstream move copies the files from this repo into distgit directly.  Maybe it doesnt matter since we dont actually merge and I can't speak to if its regenerated.

Reply:
Definitely leave it removed. The source Dockerfiles in your github repos should absolutely not have any of the OIT stuff in them. That could potentially break things.